pkg/verify/tlog.go

```diff
@@ -106,19 +103,15 @@ func VerifyArtifactTransparencyLog(entity SignedEntity, trustedMaterial root.Tru
 				if err != nil {
 					return nil, err
 				}
-
-				inclusionVerified = true
-			}
-
-			if inclusionVerified {
-				verifiedTimestamps = append(verifiedTimestamps, entry.IntegratedTime())
 			}
+			verifiedTimestamps = append(verifiedTimestamps, entry.IntegratedTime())
```

Comment:
Ah, so we don't want to append to `verifiedTimestamps` in the `if !entry.HasInclusionPromise() && !entity.HasInclusionProof()` case; we want to make sure at least one of these is true, so I think we still need `inclusionVerified` around this part.

Reply:
@steiza Good catch! Instead of adding `inclusionVerified` back, I fixed the TODO to check that one of these is true, since it would be an invalid entry if neither was.

I added two tests to catch this, one that creates an invalid entry without a proof or promise and one without any entries.